gameDriver.py

```diff
@@ -0,0 +1,187 @@
+import scrabbler as sc
+import random
+import time
+
+LETTERS = """\
+AAAAAAAAAB\
+BCCDDDDEEE\
+EEEEEEEEEF\
+FGGGHHIIII\
+IIIIIJKLLL\
+LMMNNNNNNO\
+OOOOOOOPPQ\
+RRRRRRSSSS\
+TTTTTTUUUU\
+VVWWXYYZ??\
+"""
+
+ALREADY_ENDGAME = False
+
+def get_full_bag():
+    """Returns a list of letters in the whole bag."""
+    return list(LETTERS)
+
+
+def generate_rack_and_bag(rack, bag):
+    """Randomly chooses tiles from bag and places in rack"""
+    for i in range(7 - len(rack)):
+        # If bag has ended then end game begins (as of right now this 
+        # doesn't formally mean anything, just print statement)
+        if not bag:
+            global ALREADY_ENDGAME
+            if not ALREADY_ENDGAME:
+                print('|||||||||||||||||||| END GAME STARTS NOW ||||||||||||||||||||')
+                ALREADY_ENDGAME = True
+            return rack, bag
+        new_tile = random.choice(bag)
+        rack.append(new_tile)
+        bag.remove(new_tile)
+
+    return rack, bag
+
+
+def main():
+    # List of letters we can still pick from.
+    bag = get_full_bag()
+
+    # Rack starts out empty.
+    player1Turn = True
```

Comment:
nit - it looks like this repo exclusively uses `snake_case` for variables and function names. Can you keep consistency with that style and make sure we're only using `snake_case` as well?